src/Controls/tests/BindingSourceGen.UnitTests/IntegrationTests.cs

```diff
@@ -39,7 +39,7 @@ internal static partial class GeneratedBindableObjectExtensions
             
                     {{BindingCodeWriter.GeneratedCodeAttribute}}
                     [InterceptsLocationAttribute(@"Path\To\Program.cs", 3, 7)]
-                    public static void SetBinding1(
+                    public static void SetBinding{{BindingGenerationUtilities.ComputeSha256Hash("Path\\To\\Program.cs/GeneratedBindableObjectExtensions-3-7.g.cs")}}(
```

Comment:
Nit: let's move all the `id` into separate variables, this is unnecessarily messy.